.github/workflows/extrinsic-ordering-check-from-bin.yml

```
@@ -10,7 +10,7 @@ on:
         required: true
       binary_url:
         description: A url to a Linux binary for the node containing the runtime to test
-        default: https://github.com/paritytech/cumulus/releases/download/statemine_v3/polkadot-collator
+        default: https://github.com/paritytech/cumulus/releases/download/statemine_v3/polkadot-parachain
```

https://releases.parity.io/cumulus/polkadot-v0.9.21/polkadot-parachain
```suggestion
        default: https://releases.parity.io/cumulus/polkadot-v0.9.21/polkadot-parachain
```

---

Can we pull the version from a github variable somehow? Worst case we could 'extract' it from the Cargo.lock file... don't really want to change it every time.

---

There is no need, this is just the default to help the user. It is nice to keep it close to the "good" value but no need to make that part over complicated, and definitely no need to bump that with every version. In the end, this is just a URL to the binary. I just took the chance to edit this here as the github url **cannot** work and since we rename the binary, having a pattern close to the reality for the S3 url is helpful when we call this GHA.